core/include/mapbox/geojsonvt_custom_tags.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "../../../src/data/properties.h"
+
+#include <memory>
+
+namespace mapbox {
+namespace util {
+namespace geojsonvt {
+
+struct Tags {
```

Comment:
Is this to not use geojson-vt's `std::map` implementation of `Tags`? Or there are other reasons to defining our own `Tags` structure?